index.js

```diff
@@ -13,6 +13,7 @@ const SUPPORTED_SCOPES = [
 ];
 
 module.exports.activate = function () {
+	startWorker();
 	install('linter-xo');
```

Comment:
`linter-eslint` does this work in a [`window.requestIdleCallback`](https://github.com/AtomLinter/linter-eslint/blob/master/src/main.js#L20-L32), probably to improve startup times?

I might put together another PR doing something similar

Reply:
> probably to improve startup times?

Yup